src/types.h

```diff
@@ -0,0 +1,7 @@
+#include<stdint.h>
+
+typedef struct double_ans_t {
+  double *ans;
+  uint8_t status;   // 0:ok, 1:message, 2:warning, 3:error; unix return signal: {0,1,2}=0, {3}=1
+  char message[4][256]; // STDOUT: output, STDERR: message, warning, error
```

Comment:
@mattdowle Currently this field is only used for storing errors and warnings, which won't exceed 255 chars. For storing console output for `verbose=T` (so verbose can work from parallel region) we will need to extend size of that char. Ideally we need to make it dynamic. For this PR this is enough as `verbose=T` disables topmost parallelism, and nested parallelism (where applicable) is not using `verbose` from parallel region. The cost of that is that verbose does not print in real-time, as opposed to `fread` which uses sophisticated method for printing safely and still do parallel processing.